### DISCUSSION TITLE
Create jails of existing jails or clone

### DISCUSSION THREAD

**Discussion by rubiktubik** [Jip-Hop/jailmaker#42] [Ideas]
A good feature would be a clone of an existing jail.

So I can create a jail with docker installed for example and quickly clone this one to create other jails with docker installed.

I don't know if this is possible with systemd-nspawn but this would be nice

**Comment by Jip-Hop** [comment]
There are some similar discussions:

https://github.com/Jip-Hop/jailmaker/discussions/21

https://github.com/Jip-Hop/jailmaker/discussions/22

https://github.com/Jip-Hop/jailmaker/discussions/40

**Comment by Jip-Hop** [comment]
I'm locking this discussion as I've opened dedicated issues to track feature requests. Please see [Create jails of existing jails or clone](https://github.com/Jip-Hop/jailmaker/issues/83).
